### PR TITLE
Adding system tests for nidcpower

### PR DESCRIPTION
**Pull request by injaleea**
[X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

[X] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.

### What does this Pull Request accomplish?

More syste tests for nidcpower.

- test_config_aperture_time
- test_fetch_multiple
- test_measure_multiple
- test_measure
- test_query_max_current_limit
- test_query_max_voltage_level
- test_query_min_current_limit
- test_query_output_state

### What testing has been done?

Adding all of them as commented tests. since it requires channels . Will enable them once after PR#436 merged and #444 fixed

### CODE REVIEW

File: src/nidcpower/system_tests/test_system_nidcpower.py [hunk 1]
```diff
@@ -37,3 +37,79 @@ def test_get_error(session):
...
+
+
+TODO: (Jaleel) Following tests need to be validated once issue $428,#371 fixed : Error -1074134972  (Hex 0xBFFA0044) Channel or or repeated capability name required
+def test_measure(session):
+    session.source_mode = nidcpower.SourceMode.SINGLE_POINT
+    session.output_function = nidcpower.OutputFunction.DC_VOLTAGE
+    session.voltage_level_range = 6
+    session.voltage_level = 2
+    with session.initiate():
+        reading = session.measure(1) ## enum missing from enum.py. Shouldn't be an enum ?
```
Comment (Fladolcetta) [inline src/nidcpower/system_tests/test_system_nidcpower.py]: What does this mean?

File: src/nidcpower/system_tests/test_system_nidcpower.py [hunk 2]
```diff
@@ -39,16 +39,19 @@ def test_get_error(session):
...
-'''
-TODO: (Jaleel) Dependent on PR#439 and Issue $428,#371 : Error -1074134972  (Hex 0xBFFA0044) Channel or or repeated capability name required
 def test_config_aperture_time(session):
-    session.configure_aperture_time(2, nidcpower.ApertureTimeUnits.POWER_LINE_CYCLES)
-    session.configure_aperture_time(0.01666, nidcpower.ApertureTimeUnits.SECONDS)
-    assert session.aperture_time_units == nidcpower.ApertureTimeUnits.SECONDS
-    assert session.aperture_time == 0.01666
+    with nidcpower.Session('FakeDevice', '0', False, 'Simulate=1, DriverSetup=Model:4143; BoardType:PXIe') as session:
+        session.configure_aperture_time(2, nidcpower.ApertureTimeUnits.POWER_LINE_CYCLES)
+        session.configure_aperture_time(0.01666, nidcpower.ApertureTimeUnits.SECONDS)
```
Comment (Fladolcetta) [inline src/nidcpower/system_tests/test_system_nidcpower.py]: Why do we call the same function twice? Should the first one be removed?

**Comment by Fladolcetta** [comment]
One comment, can you add the commented out enum tests to #454 and remove them from here?

File: src/nidcpower/system_tests/test_system_nidcpower.py (hunk 2 above)
Comment (injaleea) [reply src/nidcpower/system_tests/test_system_nidcpower.py]: Its to confirm the attributes are set properly by calling twice with different parameter and validating latest one is set properly

Comment (Fladolcetta) [reply src/nidcpower/system_tests/test_system_nidcpower.py]: A better test might be to do the following, where we assert against defaults. Change to non-default, then assert that it changed:

assert session.aperture_time_units == nidcpower.ApertureTimeUnits.POWER_LINE_CYCLES (make this match the default)
assert session.aperture_time == 2 (make this match the default)
session.configure_aperture_time(0.01666, nidcpower.ApertureTimeUnits.SECONDS)
assert session.aperture_time_units == nidcpower.ApertureTimeUnits.SECONDS
aperture_time = session.aperture_time
expected_aperture_time = 0.01666
....

Comment (injaleea) [reply src/nidcpower/system_tests/test_system_nidcpower.py]: fixed

**Pull request merged**